src/giftCards/GiftCardUpdatePage/GiftCardUpdatePageHeader/GiftCardEnableDisableSection.tsx

```diff
@@ -0,0 +1,105 @@
+import useNotifier from "@saleor/hooks/useNotifier";
+import { commonMessages } from "@saleor/intl";
+import { ConfirmButton } from "@saleor/macaw-ui";
+import commonErrorMessages from "@saleor/utils/errors/common";
+import classNames from "classnames";
+import React, { useContext, useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+
+import { GiftCardDetailsContext } from "../providers/GiftCardDetailsProvider";
+import { giftCardEnableDisableSectionMessages as messages } from "./messages";
+import {
+  useGiftCardActivateMutation,
+  useGiftCardDeactivateMutation
+} from "./mutations";
+import { useGiftCardEnableDisableSectionStyles as useStyles } from "./styles";
+import { GiftCardActivate } from "./types/GiftCardActivate";
+import { GiftCardDeactivate } from "./types/GiftCardDeactivate";
+
+const GiftCardEnableDisableSection: React.FC = () => {
+  const classes = useStyles({});
+  const notify = useNotifier();
+  const intl = useIntl();
+
+  const {
+    giftCard: { id, isActive }
+  } = useContext(GiftCardDetailsContext);
```

Comment:
Weren't we supposed to use `useGiftCard` hooks for that?

Reply:
Yes, leftover because this was done before we discussed using hooks in the other pr. Changing